statics/src/test/java/integration/ScreenshotsTest.java

```diff
@@ -52,4 +52,15 @@ void canTakeScreenshotAsTemporaryFile() throws IOException {
       .hasSize(1);
     assertThat(screenshots.getContextScreenshots().get(0)).isSameAs(screenshot);
   }
+
+  @Test
+  void canTakeFullScreenshot() throws IOException {
```

Comment:
What this test supposed to test? I changed `WebdriverPhotographer` to old default implementation without full screens support and test won't fail on my pc. I supposed it should fail if we don't take full screen screenshot. What am I missing?
![image](https://user-images.githubusercontent.com/6514276/172591036-5ab34aea-5e2b-4bff-82f7-88b616802335.png)

Reply:
Yes, these checks for size a crap. Need to review them.

Reply:
@BorisOsipov Valid point. Fixed in https://github.com/selenide/selenide/pull/1858 (old tests check the old logic, new tests check the new logic).